src/util/simplify_expr.cpp

```diff
@@ -637,7 +637,7 @@ simplify_exprt::simplify_typecast(const typecast_exprt &expr)
     op_type.id() == ID_pointer && expr.op().operands().size() == 1)
```

Comment:
We should get rid of ` && expr.op().operands().size() == 1`

Reply:
done